### ISSUE TITLE
core module ships with wrong maven scope

### ISSUE THREAD

**Issue by jmkgreen** [OpenFeign/feign#508]
Added `feign-httpclient` and `feign-jackson` to our `pom.xml` yet unable to get `feign.RequestLine` (etc) annotations.

It seems `feign-core` with these annotations ships with `runtime` scope, not `compile` scope?

**Comment by jmkgreen** [comment]
Right I have been told that I'm no longer supposed to search `search.maven.org` for `feign` and take the ones `com.netflix.feign` but `io.github.openfeign` instead. Problem solved.

Perhaps updating the README to provide developers with the correct Maven co-ordinates would be wise?

**Comment by dsyer** [comment]
If you search for "feign" you will find that the most recent releases come from the `io.github.openfeign` group id. I don't think that's very hard to follow actually. But I wouldn't mind seeing a copy-paste maven/gradle Getting Started section in the README either. IMO the original point about runtime scope is some kind of red herring (a jar file does not ship with a scope attached to it).

**Comment by jmkgreen** [comment]
I'm afraid a search for `feign` only showed netflix-packaged versions on the home screen with reasonable (to me) last published dates. It did not cross my mind that I was looking at old results.

Nor, frankly, did I even notice the GitHub url was now `OpenFeign`.

**Comment by adriancole** [comment]
sorry about the confusion.. org change has some collateral damage and this
is an example of it

**Comment by kdavisk6** [comment]
This has been addressed as part of the org change.  
Closing, marking this issue as stale.  If this issue still exists, please open a new issue.
